ibis/backends/tests/sql/test_compiler.py

```diff
@@ -230,3 +231,15 @@ def test_union_order_by(snapshot):
     expr = t.order_by("b").union(t.order_by("b"))
     snapshot.assert_match(to_sql(expr), "out.sql")
     assert_decompile_roundtrip(expr, snapshot, eq=schemas_eq)
+
+
+def test_window_with_row_number_compiles(snapshot):
```

Comment:
A couple things:

1. This doesn't seem like the correct place for this test since it's not actually varying the test over the backends.
2. `snapshot` isn't being used, did you mean to use it?

Reply:
1. Agree, though where should I test it then? Under `ibis/backends/base/sqlglot/tests`?
2. Yes, I intended to use the snapshot fixture, but forgot it eventually. W

Reply:
`ibis/backends/base/sqlglot/tests` seems fine

Reply:
Moved.